android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -71,17 +71,6 @@
 	public static final int RETURNKEY_DEFAULT = 9;
 	public static final int RETURNKEY_SEND = 10;
 
-	private static final int KEYBOARD_ASCII = 0;
-	private static final int KEYBOARD_NUMBERS_PUNCTUATION = 1;
-	private static final int KEYBOARD_URL = 2;
-	private static final int KEYBOARD_NUMBER_PAD = 3;
-	private static final int KEYBOARD_PHONE_PAD = 4;
-	private static final int KEYBOARD_EMAIL_ADDRESS = 5;
-	@SuppressWarnings("unused")
-	private static final int KEYBOARD_NAMEPHONE_PAD = 6;
-	private static final int KEYBOARD_DEFAULT = 7;
-	private static final int KEYBOARD_DECIMAL_PAD = 8;
-
 	// UIModule also has these as values - there's a chance they won't stay in sync if somebody changes one without changing these
 	private static final int TEXT_AUTOCAPITALIZATION_NONE = 0;
```

Comment:
We should probably remove these constants too, and use `UIModule`.

Reply:
Same for `RETURNKEY_*` constants

Reply:
Removed both blocks and pointed the constant to the UIModule namespace.